test/util/test_cd_string_commands.vader

```diff
@@ -8,11 +8,21 @@ After:
 
 Execute(CdString should output the correct command string):
   " We will check that escaping is done correctly for each platform.
-  AssertEqual
-  \ has('unix') ? 'cd ''/foo bar/baz'' && ' : 'cd "/foo bar/baz" && ',
-  \ ale#path#CdString('/foo bar/baz')
+  if has('win32')
+    AssertEqual
+    \ 'cd /d "/foo bar/baz" && ',
+    \ ale#path#CdString('/foo bar/baz')
+  else
+    AssertEqual
+    \ has('unix') ? 'cd ''/foo bar/baz'' && ' : 'cd "/foo bar/baz" && ',
```

Comment:
This was essentially a negative win32 check. Do you mind going back to this one and adding the `/d` flag to the RHS, rather than the multiline `if/else`?

Reply:
OK, I will fix it.